index/rolodex.go

```diff
@@ -159,14 +160,14 @@ func (r *Rolodex) SetRootNode(node *yaml.Node) {
 
 func (r *Rolodex) AddExternalIndex(idx *SpecIndex, location string) {
 	r.indexLock.Lock()
+	r.indexes = append(r.indexes, idx)
```

Comment:
This was the bug it took me so long to find. The remote loader uses `AddExternalIndex` to add the index of any remote file to the rolodex. Previously, this wouldn't get the index added to `r.indexes`. When bundling, `rolodex.GetIndexes` would get called but not return the indices of remote files. That's why any refs made from remote files would not get added. 🤦

Reply:
Ah! man, I know how mad this code is and how complex it can get when chasing down a bug in there. Well done!